misk-tailwind/config/README.md

```diff
@@ -0,0 +1,32 @@
+# Tailwind CSS
+
+Use the Tailwind CLI in this folder to generate tree-shaken CSS for Misk. 
+
+Make a copy of this folder to your service repo to generate tree-shaken CSS for your application.
```

Comment:
Do we expect things in this folder to change after this PR?

Reply:
I dug down into the code and found this (snipped to relevant parts):

```
@HtmlTagMarker
interface Tag {
    ....

    operator fun String.unaryPlus(): Unit {
        text(this)
    }

    fun text(s: String) {
        consumer.onTagContent(s)
    }

    ...
}
```

So tl;dr it's just a convenient shorthand for calling `consumer.onTagContent` (which I guess is a foundational part of the framework).